InterviewFlashCards-Bridging-Header.h

```diff
@@ -0,0 +1,16 @@
+//
+//  InterviewFlashCards-Bridging-Header.h
+//  
+//
+//  Created by Charles Kang on 10/14/16.
```

Comment:
Depends, might also apply to the classes we created, but now that I think about it, that might only make sense in the context of a company code base cause the company owns it so we might end up just leaving it.. Or we should come up with a trademark one-liner that we add to each class.